domain/src/main/java/org/oppia/domain/exploration/ExplorationRetriever.kt

```diff
@@ -12,10 +12,12 @@ import org.oppia.app.model.RuleSpec
 import org.oppia.app.model.State
 import org.oppia.app.model.StringList
 import org.oppia.app.model.SubtitledHtml
+import org.oppia.app.model.Voiceover
+import org.oppia.app.model.VoiceoverMapping
 import java.io.IOException
 import javax.inject.Inject
 
-const val TEST_EXPLORATION_ID_5 = "test_exp_id_5"
+const val TEST_EXPLORATION_ID_5 = "DIWZiVgs0km-"
```

Comment:
Why this change?

Reply:
I changed this to be the actual corresponding exploration id to the recording filenames, so I was able to test that creating the audio url works.